recipes/pyproprop/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pyproprop" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 853fd8eb12baeeb43c3bf8388f491175234ac7b582e062e8c1f3297651886915
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - numpy
```

Comment:
_pyproprop_ is currently using a version module within the package to single source the package version (see https://packaging.python.org/guides/single-sourcing-package-version/, technique #6). This means that `setup.py` requires the package to be imported in order for the version kwarg in `setuptools.setup()` to be set correctly. Importing the package also therefore imports other modules which have a _Numpy_ dependency. Is there are preferred approach for single-sourcing package versions for projects on _conda-forge_? Otherwise I will refactor _pyproprop_ to use a `VERSION.txt` in the package root directory instead so that the package doesn't need to be imported by `setup.py` and we can remove this _Numpy_ host requirement.

Reply:
Addressed in e55ec183245e1e80b851fdbba960786c974896f0 and d27cfd84f752351de68c7dee51277dee74b2c6f5 (by moving to *pyproprop* 0.1.1).